cmd/engine/main.go

```diff
@@ -212,7 +212,7 @@ func addFlags(app *cli.App) {
 			Value: func() int64 {
 				keep := defaultConf.Workers.OCI.GCKeepStorage.AsBytes(defaultConf.Root)
 				if keep == 0 {
```

Comment:
My understanding is that we change [BuildKit's 10% default]( https://github.com/moby/buildkit/blob/master/cmd/buildkitd/config/gcpolicy_unix.go#L10) here and set it to `75%`, as specified in `gc.go` . How do we know that this is a safe default? Would `50%` be more appropriate here?

Reply:
Assuming the GC correctly kicks in (fixed in https://github.com/dagger/dagger/pull/7738), then any value here should be good. I've had no issues with any value here in all my time running buildkit.

Potentially this will result in a large space usage from dagger - but IMO, this is a better result than aggressive pruning that results in poor performance.

There is ongoing work to improve the defaults by introducing new heuristics upstream, see https://github.com/dagger/dagger/issues/7711#issuecomment-2186790489 for context on our side.

cc @marcosnils